levelkdev/swapr-dapp
# chore: remove old LP position on Arbtirum check

# Summary

Removes hooks that check for the LP positions using the old $SWPR token on Arbitrum. 

![paste](https://user-images.githubusercontent.com/1926216/159063074-fbc5d951-a7a6-4098-99d4-b0dc193e6e13.png)


# To Test

1. Visit the dapp
2. Open network tab 
3. Observe the requests, there should be no requests to the Arbitrum subgraph
